lab-joy/src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
whatever tag comes after HTML needs to be indented, so everything inside of HTML needs to be actually **inside** of HTML